electrum/gui/qml/qetxfinalizer.py

```diff
@@ -864,6 +852,9 @@ def update(self):
             self.warning = str(e)
             return
 
+        child_feerate = fee / self._new_tx.estimated_size()
+        self.feeRate = f'{child_feerate:.1f}'
```

Comment:
Would it be possible to parameterise the feerate precision?
As in, actually it is already parameterised here:
https://github.com/spesmilo/electrum/blob/bd7adc83b325e9d471c4dcffd2a73c956c9978ba/electrum/util.py#L793-L797

Could you just call `format_fee_satoshis` instead?

Reply:
Yeah sure, this was a remnant code snippet taken from the desktop client initially (probably still in there too)